FxA/FxA/RSAKeyPair.m

```diff
@@ -145,10 +151,14 @@ - (id) initWithModulus: (CHNumber*) n publicExponent: (CHNumber*) e;
         BIGNUM *_e = BN_dup([e bigNumValue]);
 
         if (_n == NULL || _e == NULL) {
+            BN_free(_n);
+            BN_free(_e);
             return nil;
         }
         _rsa = RSA_new();
         if (RSA_set0_key(_rsa, _n, _e, NULL) != 1) {
+            BN_free(_n);
+            BN_free(_e);
             return nil;
```

Comment:
Do we not also need `RSA_free(_rsa)` here?

Reply:
No, `_rsa` is a private property on this class.